packages/core/addon/utils/request.ts

```diff
@@ -166,14 +167,16 @@ export function normalizeV1toV2(request: RequestV1<string>, dataSource: string):
 
   //normalize dateTime column
   requestV2.columns.push({
+    cid: nanoid(10),
```

Comment:
Does the `defaultValue` not kick in here?

Reply:
I was just copying the implementation from the default value
https://github.com/yahoo/navi/blob/f804f05c6960fe7be5cca376ab90806624b91949/packages/core/addon/models/bard-request-v2/fragments/column.ts#L16-L17